### PR TITLE
Removed dist folder and old instructions from README after local testing

### PR DESCRIPTION
**Pull request by chrisli30**
I tested code-gen commands and docker compose up from the start, and they all work pretty great! 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,14 +92,13 @@ To ensure the SDK is functioning correctly, we have a comprehensive test suite.
    npm run test:select -- <authWithSignature>
    ```
 
-   Note: to generate an api for `TEST_API_KEY`, bring up the docker compose and run `npm run gen-api-key`.
```
Comment (chrisli30) [inline README.md:94]: There was a typo in the `gen-apikey` of package.json scripts.

**Review by chrisli30 — COMMENTED** [review]
Dont’t know why the `grpc_codegen/compiled.d.ts` file is generated, but it doesn’t show up from the `gen-protoc` script any more.

The dist folder was used to preserve manually edits of the avs_pb.d.ts file. Since they can be auto-generated I don’t think we need to keep them in the code repo. I suspect that there was a new version that fixed the code generation process.

File: package.json [hunk 2]
```diff
@@ -22,10 +22,10 @@
     "node": ">=20.18.0"
   },
   "scripts": {
-    "proto-download": "curl -s https://raw.githubusercontent.com/AvaProtocol/EigenLayer-AVS/new-task-data-structure/protobuf/avs.proto > grpc_codegen/avs.proto",
+    "proto-download": "curl -s https://raw.githubusercontent.com/AvaProtocol/EigenLayer-AVS/main/protobuf/avs.proto > grpc_codegen/avs.proto",
     "gen-protoc": "grpc_tools_node_protoc --js_out=import_style=commonjs,binary:./grpc_codegen/ --grpc_out=grpc_js:./grpc_codegen/ --plugin=protoc-gen-ts=./node_modules/.bin/protoc-gen-ts --ts_out=grpc_js:./grpc_codegen --proto_path=./grpc_codegen grpc_codegen/avs.proto",
     "build": "tsup src/index.ts --format cjs,esm --dts",
-    "gen-apikey": "docker compose exec aggregator /ava  create-api-key --role=admin --subject=apikey",
+    "gen-apikey": "sh -c 'API_KEY=$(docker compose exec aggregator /ava create-api-key --role=admin --subject=apikey) && echo \"Generated API key for tests: $API_KEY\" && echo \"Writing to .env.test to replace the TEST_API_KEY env variable\" && sed -i \"\" -e \"s/^TEST_API_KEY=.*/TEST_API_KEY=$API_KEY/\" .env.test && echo \"Done.\"'",
```
Comment (chrisli30) [inline package.json:28]: Made this script to write into .env.test to replace TEST_API_KEY’s value

File: package.json [hunk 3]
```diff
@@ -51,6 +51,7 @@
     "@types/lodash": "^4.17.12",
     "babel-jest": "^29.7.0",
     "grpc_tools_node_protoc_ts": "^5.3.3",
+    "grpc-tools": "^1.12.4",
```
Comment (chrisli30) [inline package.json:54]: Include this library in this repo. It was required in the README as a global install, but including it here makes the README instructions shorter.

File: README.md [hunk 4]
```diff
@@ -91,14 +92,13 @@ To ensure the SDK is functioning correctly, we have a comprehensive test suite.
...
+   > Note: In order to individually test `cancelTask` or `deleteTask`, `createTask` test needs to run first.
```
Comment (v9n) [inline README.md:95]: We don't need this. The test can run in any order now.

**Review by v9n — APPROVED** [review]

**Pull request merged**